iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/json/ScalingParametersDeserializer.java

```diff
@@ -67,6 +68,7 @@ public ScalingParameters deserialize(JsonParser parser, DeserializationContext c
                     parser.nextToken();
                     parameters.setPriority(JsonUtil.readValue(context, parser, ScalingParameters.Priority.class));
                 }
+                case "ignoredInjectionIds" -> parameters.setIgnoredInjectionIds(new HashSet<>(JsonUtil.parseStringArray(parser)));
```

Comment:
`ScalingParameters.VERSION` should be changed to `1.2` and a check on the minimum version should be added here:

```java
JsonUtil.assertGreaterOrEqualThanReferenceVersion(CONTEXT_NAME, "Tag: ignoredInjectionIds", version, "1.2");
```